<launch>
<arg name="robot_num"  default="2"/>
<node pkg="robo_vision" type="read_camera" name="read_camera" output="screen"/>
<node pkg="robo_vision" type="armor_detect" name="armor_detect" output="screen">
<param name="robot_num" value="(arg robot_num)"/>
</node>
</launch>
